go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/riferrei/srclient v0.5.4
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/libocr v0.0.0-20230802221916-2271752fa829
+	github.com/smartcontractkit/sqlx v1.3.5-0.20210805004948-4be295aacbeb
```

Comment:
ideally we don't inject sql dependencies - but it seems unavoidable at the current stage until the persistence layer work is done 🤔

Reply:
correct me if I'm wrong, but introducing having `pg.Opts` requires importing a lot of extra files/types? (like uint32.go or json.go)

Reply:
@aalu1418 yup that is right. Introducing `pg.Opts` require us to import extra file types. Do you think we should seperate them instead? i.e generalise pg.Opts

Reply:
agreed with how we should not inject sql dependencies. This PR is better lived in a branch for now